alpaca/entities.go

```diff
@@ -159,7 +160,7 @@ type AccountActivity struct {
 	Symbol          string          `json:"symbol"`
 	LeavesQty       decimal.Decimal `json:"leaves_qty"`
 	CumQty          decimal.Decimal `json:"cum_qty"`
-	Date            time.Time       `json:"date"`
+	Date            civil.Date      `json:"date"`
```

Comment:
This is technically a backward-incompatible change. However, if the original code never worked (??? is this really the case??) and the new one works with all possible account activities that the server may send, then it could be fine.

Reply:
The original code in the sdk never worked. Server-side code never changed.